core/commands/refs.go

```diff
@@ -313,6 +313,8 @@ func (rw *RefWriter) WriteEdge(from, to key.Key, linkname string) error {
 
 	var s string
 	switch {
+	case rw.PrintFmt == "<dst>":
```

Comment:
yeah, i'd call this a premature op. The added time of going through the other case will be dwarfed by the cost of a b58 marshal and api latency